### ISSUE TITLE
Old Jenkins version

### ISSUE THREAD

**Issue by Svet1395** [asmagin/jenkins-on-windowsservercore#7]
could you update to the latest jenkins version, please?

**Comment by Svet1395** [comment]
Ideally if you could a small script to have jenkins automatically update itself. The old version is causing a lot of issues with plugins.

**Comment by xcellenceit** [comment]
Any update on this?

**Comment by asmagin** [comment]
I do not believe that I'll get back to this project in the nearest future as I'm not using it now, but I can check and merge contributions.
